### PR TITLE
Improve Grammar for Authentication strategies

### PR DESCRIPTION
**Pull request by chaosaffe**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1685)

### CODE REVIEW

**Comment by deads2k** [comment]
No change to the tech content.  I'll leave it up for doc review.

**Comment by chaosaffe** [comment]
I signed it!

Contributing under the CNCF CLA for Loodse GmbH

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ or be treated as an anonymous user.
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
 authenticate API requests through authentication plugins. As HTTP request are
```
Comment (devin-donnelly) [inline docs/admin/authentication.md]: request -> requests

**Comment by devin-donnelly** [comment]
Just one nit (though it's not in the part you actually changed, the sentence doesn't read correctly without it).

**Pull request merged**